org.adempiere.base/src/org/compiere/model/MOrder.java

```diff
@@ -75,6 +75,8 @@
 	 */
 	private static final long serialVersionUID = -7784588474522162502L;
 
+	private static final boolean isReactWHOrderReverseInvoice = MSysConfig.getBooleanValue(MSysConfig.REACTIVATE_WH_ORDER_REVERSES_INVOICE, false, Env.getAD_Client_ID(Env.getCtx()));
```

Comment:
@mschnbeck - just for showing best practice - but please don't fix the pull request as we're going to discuss it better in the ticket:
- MSysConfig.get as a static is not a good idea as it can be different per every client, or it can be changed and cache reset.

Reply:
Ok. I got the idea from another usage of MSysConfig but maybe of course that was installation specific.